cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -102,6 +103,8 @@ ocv_optimization_process_obsolete_option(ENABLE_NEON NEON OFF)
 
 ocv_optimization_process_obsolete_option(ENABLE_VSX VSX ON)
 
+ocv_optimization_process_obsolete_option(ENABLE_RISCV RISCV OFF)
```

Comment:
`ENABLE_RISCV` is bad choise as it's general architecture name. I propose to use something `ENABLE_RVV` or enable `ENABLE_RISCV_VEXT`.